Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -45,6 +45,7 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
                 cell.imageView?.sd_setImageWithURL(NSURL(string: favicon.url)!, placeholderImage: profile.favicons.defaultIcon)
             }
             cell.textLabel?.text = bookmark?.title
+            cell.accessibilityLabel = bookmark?.title
```

Comment:
This is unnecessary, I think -- the accessibility label defaults to the text in the textview.